### DISCUSSION TITLE
Compatibility with mod-individual-progression?

### DISCUSSION THREAD

**Discussion by Badgermilk0** [mod-playerbots/mod-playerbots#519] [General]
Hi,

I'm curious if there is any documented compatibility issues when using this Azerothcore branch together with mod-playerbots and -mod-individual-progression: github.com/ZhengPeiRu21/mod-individual-progression

Also if there are comments from people that have tried this combination and if there was issues you had to deal with.

Thanks!

//Badger

**Comment by Kyroth88** [comment]
A bunch of us in discord are using this combo and I have been enjoying it, I have no significant issues I have delt with.
